unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/CharacterController/DCLCharacterController.cs

```diff
@@ -586,4 +589,9 @@ bool IsLastCollisionGround()
     {
         return (lastCharacterControllerCollision & CollisionFlags.Below) != 0;
     }
+
+    public void ForceReportMovement()
```

Comment:
Is it used? Nonetheless i would recommend making `ReportMovement()` function public